components/locales/ru-RU.json

```diff
@@ -13,19 +13,28 @@
     "next_3": "Следующие 3"
   },
   "DatePicker": {
+    "firstDayOfWeek": "Понедельник",
```

Comment:
this property is needed to be English to specific the first day of week. Like Sunday or Monday.

Reply:
Got it!